### PR TITLE
fix(profile): 'Profile page bug fixes'

### PR DESCRIPTION
**Pull request by abhishek-mishra019**
Signed-off-by: Abhishek Mishra [am1426620@gmail.com](mailto:am1426620@gmail.com)
### Functionality:
to fix the ui and loading issues of profile page

### Solution:
fixed the UI and date formats, data representation for new User data, and loading issues

### Risk level:
- [x] high 
- [ ] medium
- [ ] low

### How to test:
<-- please tell us how to test functionality/bug -->

### CODE REVIEW

File: src/app/body/profile/profile.component.ts [hunk 1]
```diff
@@ -217,13 +221,15 @@ export class ProfileComponent implements OnInit {
       this.website = this.authService.userAppSetting.Website;
       if (this.website.includes("https://") == false) {
         this.website = "https://" + this.website;
+        // this.website= 'github.com'
```
Comment (sanjaykrishna1203) [inline src/app/body/profile/profile.component.ts:224]:
```suggestion
      this.website = this.authService.userAppSetting.Website;
      if (this.website.includes("https://") == false && this.website != "") {
        this.website = "https://" + this.website;
        // this.website= 'github.com'
```

File: src/app/body/profile/profile.component.html [hunk 2]
```diff
@@ -43,54 +43,57 @@
...
-                            <span class="card-title my-4" *ngIf="managerEmail">
+                            <span class="card-title my-4" >
                                 <app-profile-icons [profileIcon]="'building'"></app-profile-icons>
                                 Managed by {{ managerEmail }}
                             </span>
                         </div>
                     </div>
                     
-                    <div class="row py-2">
+                    <div class="row py-2" *ngIf="website != 'https://'">
```
Comment (sanjaykrishna1203) [inline src/app/body/profile/profile.component.html:83]:
```suggestion
                    <div class="row py-2" *ngIf="website">
```

**Review by sanjaykrishna1203 — CHANGES_REQUESTED** [review]

File: src/app/body/profile/profile.component.ts (hunk 1 above)
Comment (abhishek-mishra019) [reply src/app/body/profile/profile.component.ts]: committed the changes

**Review by sanjaykrishna1203 — APPROVED** [review]

**Review by viv9k — APPROVED** [review]
Approved

**Pull request merged**